core/src/main/java/org/apache/calcite/sql/type/ReturnTypes.java

```diff
@@ -410,6 +410,24 @@ public static SqlCall stripSeparator(SqlCall call) {
   public static final SqlReturnTypeInference DOUBLE_FORCE_NULLABLE =
       DOUBLE.andThen(SqlTypeTransforms.FORCE_NULLABLE);
 
+  /**
```

Comment:
this comment seems wrong too

Reply:
Moved the comment to `DECIMAL_OR_DOUBLE`. Removed `DOUBLE_OR_DECIMAL`.